lib/p2p/Pool.ts

```diff
@@ -135,6 +137,25 @@ class Pool extends EventEmitter {
     this.connected = false;
   }
 
+  private verifyReachability = () => {
+    this.handshakeData.addresses!.forEach(async (address) => {
+      const externalAddress = `${address.host}:${address.port}`;
```

Comment:
There's an `addressUtils.toString()` method that does this. It does the same thing, but we might as well use that here.